src/mono/wasm/runtime/imports.ts

```diff
@@ -39,12 +39,13 @@ export function set_imports_exports(
     runtimeHelpers.requirePromise = imports.requirePromise;
 }
 
-export const runtimeHelpers: RuntimeHelpers = <any>{
-    javaScriptExports: {},
+const initialRuntimeHelpers: Partial<RuntimeHelpers> =
+{
+    javaScriptExports: {} as any,
     mono_wasm_load_runtime_done: false,
     mono_wasm_bindings_is_ready: false,
-    max_parallel_downloads: 16,
+    maxParallelDownloads: 16,
```

Comment:
What does the style difference between this, and other fields mean?

Reply:
This fix of a is bad merge,  when I renamed stuff last week. This is what broke throttling.

Reply:
we could rename the rest of them at some point. They are all internal.